int32 id
int32 left_id
int32 right_id
geometry_msgs/Point[] center
geometry_msgs/Point[] left_boundary
geometry_msgs/Point[] right_boundary